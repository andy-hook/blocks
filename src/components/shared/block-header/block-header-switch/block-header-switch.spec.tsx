import React from "react"
import BlockHeaderSwitch from "./block-header-switch"
import { renderWithTheme } from "@test-utils"

jest.mock("react-truncate-string")

describe("<BlockHeaderSwitch />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      "dark",
      <BlockHeaderSwitch
        onDetailsClock={jest.fn}
        onTransactionsClick={jest.fn}
      />
    )
    expect(tree).toMatchSnapshot()
  })
})
