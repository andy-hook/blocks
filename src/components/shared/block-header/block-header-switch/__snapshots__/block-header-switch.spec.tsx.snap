// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockHeaderSwitch /> renders correctly 1`] = `
<div
  className="sc-bdVaJa bDWFJH"
>
  <button
    className="sc-bwzfXH dcmFEq"
    onClick={[Function]}
  >
    <div
      className="sc-htpNat gRXXol"
    >
      Details
    </div>
  </button>
  <button
    className="sc-bwzfXH jHohNU"
    onClick={[Function]}
  >
    <div
      className="sc-htpNat gRXXol"
    >
      Transactions
    </div>
  </button>
</div>
`;
